File: frontend/src/constants/OrganizationRequisite.ts
import { Dayjs } from "dayjs";

export type OrganizationRequisite = {
  id: number;
  organization_id: number; // Main table foreign key
  payment_account: string;
  bank: string;
  bik: string;
  created_at: Dayjs;
  updated_at: Dayjs;
  created_by: number;
  updated_by: number;
};

export type OrganizationRequisiteCreateModel = {
  id: number;
  organization_id: number;
  payment_account: string;
  bank: string;
  bik: string;
};

File: backend/Domain/Entities/OrganizationRequisite.cs
namespace Domain.Entities
{
    public class OrganizationRequisite : BaseLogDomain
    {
        public int Id { get; set; }
        public string? PaymentAccount { get; set; }
        public string? Bank { get; set; }
        public string? Bik { get; set; }
        public int OrganizationId { get; set; }
    }
}
